<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storereport</class>
 <widget class="QWidget" name="storereport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_20" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <widget class="QDateEdit" name="store_report_enddateEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>110</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>总库饲料药品出入库汇总</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>60</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>总库饲料药品出入库汇总</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="storereport_startdateEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>110</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="storereport_export_btn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>0</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>分库饲料药品用料汇总</string>
      </property>
     </widget>
     <widget class="QPushButton" name="storereport_search_btn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>60</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>总库饲料药品出入库汇总</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="store_report_tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
